### PR TITLE
[TEP-0105] Proposal for removing Pipeline v1alpha1, mark as implementable

### PR DESCRIPTION
**Pull request by abayer**

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]

**Comment by lbernick** [comment]
Worth noting that we are not going to delete v1alpha1 Run

Also I think we should treat pipelineResources separately. Our deprecation table states that they are past the earliest date of removal but it might be worth giving people more of a heads up for that, and make sure we aren't using them at all first :)

**Review by lbernick — APPROVED** [review]

**Comment by lbernick** [comment]
/test pull-community-teps-lint

@pritidesai @afrittoli ok to go ahead with this?

@abayer once this accepted let's get an email announcement out as soon as we can :)

File: teps/README.md [hunk 1]
```diff
@@ -245,7 +245,8 @@ This is the complete list of Tekton teps:
 |[TEP-0102](0102-https-connection-to-triggers-interceptor.md) | HTTPS Connection to Triggers ClusterInterceptor | implementable | 2022-04-20 |
 |[TEP-0103](0103-skipping-reason.md) | Skipping Reason | implemented | 2022-05-05 |
 |[TEP-0104](0104-tasklevel-resource-requests.md) | Task-level Resource Requests | implementable | 2022-04-08 |
-|[TEP-0105](0105-remove-pipeline-v1alpha1-api.md) | Remove Pipeline v1alpha1 API | proposed | 2022-04-11 |
+|[TEP-0105](0105-remove-pipeline-v1alpha1-api.md) | Remove Pipeline v1alpha1 API | implementable | 2022-05-17 |
 |[TEP-0106](0106-support-specifying-metadata-per-task-in-runtime.md) | Support Specifying Metadata per Task in Runtime | implemented | 2022-05-27 |
 |[TEP-0107](0107-propagating-parameters.md) | Propagating Parameters | implemented | 2022-05-26 |
 |[TEP-0108](0108-mapping-workspaces.md) | Mapping Workspaces | implemented | 2022-05-26 |
+
```
[the remark this answers is not in the document]
Comment (abayer) [reply teps/README.md:252]: argh, fixing.

File: teps/0105-remove-pipeline-v1alpha1-api.md [hunk 2]
```diff
@@ -134,161 +79,20 @@ upgraded to at least `v1beta1` before `v1` is introduced.
...
-
-<!--
-Use this section to detail wether this feature needs an upgrade or
-migration strategy. This is especially useful when we modify a
-behavior or add a feature that may replace and deprecate a current one.
--->
-
-## Implementation Pull request(s)
+We will target removing the Pipeline `v1alpha1` API for the 0.38.0 release, in
+late July, 2022. This will be communicated in the release notes for 0.36.0 and 0.37.0.
```
Comment (pritidesai) [inline teps/0105-remove-pipeline-v1alpha1-api.md:85]: Are we deleting the version `v1alpha1` from the configuration?  or setting the `storage` to `false`?

https://github.com/tektoncd/pipeline/blob/main/config/300-pipeline.yaml#L27-L45

Either ways, the conversion webhook will have to be updated to not convert up/down just serve and store `v1beta1`. 

Worth being a little more explicit in the proposal. 🙏

**Review by pritidesai — APPROVED** [review]
thanks a bunch @abayer 🎉

**Review by jerop — APPROVED** [review]
🎉

**Comment by lbernick** [comment]
@abayer  please rebase
API WG: good to go once rebased

**Pull request merged**